no-vary-search.bs

```diff
@@ -102,14 +102,14 @@ The [=obtain a URL search variance=] algorithm ensures that all [=URL search var
         1. Set |result|'s [=URL search variance/vary params=] to [=URL search variance/no-vary params/wildcard=].
     1. Otherwise, if |value|["`params`"] is a [=list=]:
       1. If any [=list/item=] in |value|["`params`"] is not a [=string=], then return the [=default URL search variance=].
-      1. Set |result|'s [=URL search variance/no-vary params=] to |value|["`params`"].
+      1. Set |result|'s [=URL search variance/no-vary params=] to the result of [=parsing a key=] given each [=list/item=] in |value|["`params`"].
```

Comment:
I find this a tad confusing as apparently the result is a list and while "each" gives that away somewhat it's rather magical.

Reply:
Yeah, we don't have a good wording for `list.map(operation)`. I'll try

> to the result of applying [parse a key] to each [item] in ...